dist/svg/ds6/icons.svg

```diff
@@ -136,10 +136,10 @@
         <path d="M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12c0-3.183-1.264-6.235-3.515-8.485A11.996 11.996 0 0012 0zm0 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10zm-2-7.41l7.29-7.3h.01a1 1 0 011.41 1.41l-8 8a1 1 0 01-1.41 0l-4-4a1 1 0 011.41-1.41l3.29 3.3z"></path>
     </symbol>
     <symbol viewBox="0 0 24 24" id="icon-confirmation-filled">
-        <path fill="#36CF57" d="M0 12C0 5.373 5.373 0 12 0c3.183 0 6.235 1.264 8.485 3.515A11.996 11.996 0 0124 12c0 6.627-5.373 12-12 12S0 18.627 0 12zm10.71 4.71l8-8h-.01a1 1 0 00-1.41-1.41L10 14.59 6.71 11.3a1 1 0 00-1.41 1.41l4 4a1 1 0 001.41 0z"></path>
+        <path fill="#28A443" d="M0 12C0 5.373 5.373 0 12 0c3.183 0 6.235 1.264 8.485 3.515A11.996 11.996 0 0124 12c0 6.627-5.373 12-12 12S0 18.627 0 12zm10.71 4.71l8-8h-.01a1 1 0 00-1.41-1.41L10 14.59 6.71 11.3a1 1 0 00-1.41 1.41l4 4a1 1 0 001.41 0z"></path>
```

Comment:
I think we should eventually get away from this and use CSS classes.

Reply:
Great idea!